tesseract_core/runtime/config.py

```diff
@@ -27,10 +27,12 @@ def update_config(**kwargs: Any) -> None:
     global _current_config
 
     conf_settings = {}
-    for field in RuntimeConfig.model_fields.keys():
-        env_key = f"TESSERACT_{field.upper()}"
-        if env_key in os.environ:
-            conf_settings[field] = os.environ[env_key]
+
+    if _current_config is None:
+        for field in RuntimeConfig.model_fields.keys():
+            env_key = f"TESSERACT_{field.upper()}"
+            if env_key in os.environ:
+                conf_settings[field] = os.environ[env_key]
```

Comment:
Should this be guarded behind an option `init_from_envvars` instead? (That is `False` by default, but set to `True` within the initial `get_config`.)